context/app/static/js/components/organ/Azimuth/style.js

```diff
@@ -5,4 +5,10 @@ const StyledPaper = styled(Paper)`
   padding: 20px 40px 20px 40px;
 `;
 
-export { StyledPaper };
+const StyledLink = styled.span`
+  a {
+    color: rgb(42, 111, 184);
```

Comment:
You can use the color from the theme. ` color: ${(props) => props.theme.palette.info.main};`

Reply:
I agree; I just double checked, and this would match the figma designs more closely as well, since the links are defined as the same shade of blue that's in our theme (#2A6FB8), while the currently defined color is a little bit different (#2A3B6F).

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/93310867-6178-46be-a0df-d4f50996a34a)